packages/arb-token-bridge-ui/tests/support/commands.ts

```diff
@@ -247,6 +247,13 @@ export function findMoveFundsButton(): Cypress.Chainable<JQuery<HTMLElement>> {
     .should('be.visible')
 }
 
+export function startTransfer() {
+  cy.wait(5_000)
+  cy.findMoveFundsButton().click()
+  cy.wait(15_000)
+  cy.confirmMetamaskTransaction()
```

Comment:
Can we add longer timeout for the 2 commands rather than wait maybe?

Reply:
For the first timeout we want to make sure there are no race conditions when the button is clicked.

For the second one, it's more about the number of retries synpress attempts when looking for the metamask window. We can't modify the number of attempts so we give more time for the metamask pop up to show.